docs/changelog/98309.yaml

```diff
@@ -3,3 +3,31 @@ summary: "Integrate Elasticsearch Query Language, ES|QL"
 area: Query Languages
```

Comment:
Should we change this to ES|QL to match all other ES|QL PR's (right now this is the only ES|QL PR in `Query Languages`)?